test/blackbox-tests/test-cases/nix-result-in-tree.t/dune

```diff
@@ -0,0 +1,2 @@
+(executable
+ (name ocamlformat))
```

Comment:
I don't think this has anything to do with Nix per-se, thus I would suggest renaming it to something akin to `read-only-symlink-target.t`.